examples/fishjam-chat/ios/ScreenBroadcast/Info.plist

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
**thought:** I'm wondering if we should setup a ScreenBroadcast extension as an expo plugin or leave it to our users 🤔

Reply:
I created FCE-524 as followup